pkg/clientconfiguration/conf.go

```diff
@@ -50,7 +50,7 @@ var StaticConfigurations = []ConfigurationItem{
 		Merge: false,
 	},
 	{
-		Match: &ScriptMatch{Expr: `c.sdk == "flutter" && c.version == "2.4.2" && c.os == "android"`},
+		Match: &ScriptMatch{Expr: `c.sdk == "flutter" && c.version == "2.4.2"`},
```

Comment:
I think we don't have to disable anymore? since moving it to the end is sufficient?

Reply:
Yes, my bad. Will take out the rule.